src/lib/permissions/rules.ts

```diff
@@ -1,6 +1,58 @@
 import { rule } from 'graphql-shield';
+import { Context } from '../../context';
 
-export const isAuthenticated = rule({ cache: 'contextual' })(async (_, __, ctx) => {
-    console.log(ctx.userId);
+export const isAuthenticated = rule({ cache: 'contextual' })(async (_, __, ctx: Context) => {
     return ctx.userId !== null;
 });
+
+export const isParticipantOfMeeting = rule({ cache: 'contextual' })(async (_, { meetingId }, ctx: Context) => {
+    const participant = ctx.prisma.participant.findFirst({
+        where: {
+            userId: ctx.userId,
+            meetingId,
+        },
+    });
+    return participant !== null;
```

Comment:
Should return a ForbiddenError if null

Reply:
Shouldn't the rules just provide a boolean saying if they have permission or not, not throwing an Error when they don't have permission?

Reply:
A rule will show an error message which is "Not Authenticated" whenever a rule is returned true. If want more descriptive error messages (which would be nice, compared to Not Authenticated), then we can return an Error with the display message :)